lib/pluggy_elixir/http_adapter/response.ex

```diff
@@ -0,0 +1,17 @@
+defmodule PluggyElixir.HttpAdapter.Response do
+  @moduledoc """
+  This module defines a `PluggyElixir.HttpAdapter.Response.t/0` struct that stores http responses.
+  """
+
+  @type body :: any()
+  @type headers :: [{binary(), binary()}]
+  @type status :: integer() | nil
+
+  @type t :: %__MODULE__{
+          body: body(),
+          headers: headers(),
+          status: status()
+        }
+
+  defstruct [:body, :headers, :status]
```

Comment:
Same as the other ones, `defstruct` should be before `@type` according to [Elixir Style Guide](https://github.com/brainnco/pluggy_elixir/pull/1/files#r620294709).

What do you think about adding `@enforce_keys` with all fields to `defstruct`?